extensions/reviewed/TimeFormatter.json

```
@@ -1,22 +1,33 @@
 {
   "author": "@Bouh",
   "category": "User interface",
-  "description": "Expressions to transform time in seconds to format like HH:MM:SS. Ideal to display timers on screen.",
   "extensionNamespace": "",
   "fullName": "Time formatting",
   "helpPath": "",
   "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4iICJodHRwOi8vd3d3LnczLm9yZy9HcmFwaGljcy9TVkcvMS4xL0RURC9zdmcxMS5kdGQiPjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgdmVyc2lvbj0iMS4xIiBpZD0ibWRpLWNsb2NrLWRpZ2l0YWwiIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNMiw2QTIsMiAwIDAsMCAwLDhWMTZBMiwyIDAgMCwwIDIsMThIMjJBMiwyIDAgMCwwIDI0LDE2VjhBMiwyIDAgMCwwIDIyLDZNMiw4SDIyVjE2SDJNMyw5VjEwLjVINi4yNUwzLDE1SDQuNzVMOCwxMC41VjlNOS4yNSw5VjEwLjVIMTAuNzVWOU0xMiw5VjEwLjVIMTMuNVYxNUgxNVY5TTE3LDlBMSwxIDAgMCwwIDE2LDEwVjE0QTEsMSAwIDAsMCAxNywxNUgyMEExLDEgMCAwLDAgMjEsMTRWMTBBMSwxIDAgMCwwIDIwLDlNMTcuNSwxMC41SDE5LjVWMTMuNUgxNy41TTkuMjUsMTMuNVYxNUgxMC43NVYxMy41IiAvPjwvc3ZnPg==",
   "name": "TimeFormatter",
   "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/clock-digital.svg",
-  "shortDescription": "Expressions to transform time in seconds to format like HH:MM:SS. Ideal to display timers on screen.",
-  "version": "0.0.1",
+  "shortDescription": "Converts seconds into standard time formats, such as HH:MM:SS. ",
+  "version": "0.0.2",
+  "description": [
+    "Ideal for displaying timers on screen.",
+    "",
+    "Formats included:",
+    "HH:MM:SS",
+    "HH:MM:SS.000 (displays milliseconds)"
```

You can use a Markdown list:

```suggestion
    "Formats included:",
    "* HH:MM:SS",
    "* HH:MM:SS.000 (displays milliseconds)"
```